plugins/modules/storage/zfs/zfs.py

```diff
@@ -37,6 +37,11 @@
       - A dictionary of zfs properties to be set.
       - See the zfs(8) man page for more information.
     type: dict
+notes:
+  - C(check_mode) is supported, but in certain situations it may report a task
+    as changed that would not reported as changed if C(check_mode) were disabled.
```

Comment:
```suggestion
    as changed that will not be reported as changed if C(check_mode) were disabled.
```

Reply:
I think the two grammatically correct ways to phrase this are:

- it may report a task as changed that __would__ not reported as changed __if__ C(check_mode) __were__ disabled.
- it may report a task as changed that __will__ not __be__ reported as changed __when__ C(check_mode) __is__ disabled.

I personally prefer the first phrasing, but I'm fine with either. What do you think?

Reply:
I don't feel good with first sentence, but feel free to ignore me as I'm not a native English speaker anyway ;)

Reply:
I changed it to the second phrasing. I think that fits better with the rest of the note, since I wrote it as "__when__ the zpool altroot option is set" instead of "__if__ the zpool...".